client/landing/jetpack-cloud/components/log-item/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import CardHeading from 'components/card-heading';
+
+import './style.scss';
+
+interface Props {
+	children?: ReactNode;
+	header: string;
+	subheader: string;
+	highlight?: string;
+	tag?: string;
+}
+
+class LogItem extends React.PureComponent< Props > {
+	static defaultProps = {
+		highlight: 'info',
+		tag: 'new',
+	};
+
+	renderHeader() {
+		const { header, subheader, tag } = this.props;
+
+		return (
+			<div className="log-item__header-wrapper">
+				<span className="log-item__tag">{ tag }</span>
+				<CardHeading tagName="h2" size={ 18 }>
+					{ header }
+				</CardHeading>
+				<p className="log-item__subheader">{ subheader }</p>
```

Comment:
Should we display the `<p>` only if subheader is defined?

Reply:
Resolved in 6b85d3630a910d8432cd3141981e1eec3070b975.